src/util/system.cpp

```diff
@@ -967,10 +968,11 @@ fs::path GetPidFile()
 void CreatePidFile(const fs::path &path, pid_t pid)
```

Comment:
I don't quite understand why this doesn't return an error status, and that not being able to create the PID file is not fatal.